### ISSUE TITLE
write-manifest produces the wrong entrypoint for Shiny Express apps

### ISSUE THREAD

**Issue by aronatkins** [rstudio/rsconnect-python#540]
The deploy command understands Shiny express applications, but not the write-manifest command.

See: https://github.com/rstudio/rsconnect-python/pull/521

```python
from shiny import reactive
from shiny.express import input, render, ui
from shinywidgets import render_plotly

ui.page_opts(title="Penguins dashboard", fillable=True)

with ui.sidebar():
    ui.input_selectize(
        "var", "Select variable",
        ["bill_length_mm", "bill_depth_mm", "flipper_length_mm", "body_mass_g", "year"]
    )
    ui.input_numeric("bins", "Number of bins", 30)

@reactive.calc
def df():
    from palmerpenguins import load_penguins
    return load_penguins()[input.var()]

with ui.card(full_screen=True):
    @render_plotly
    def hist():
        import plotly.express as px
        p = px.histogram(df(), nbins=input.bins())
        p.layout.update(showlegend=False)
        return p
```

```
shiny==0.7.0
shinywidgets==0.3.0
```

The rsconnect-python 1.22.0 release creates a manifest with an entrypoint of `app`.

**Comment by aronatkins** [comment]
The entrypoint in the manifest should be something like: `shiny.express.app:app_2e_py`

**Comment by wch** [comment]
This has been fixed by #558.
